src/templates/Dashboard.vue

```diff
@@ -173,6 +171,7 @@ export default {
           entity.status = "No Previous Check-in"
         }
       }
+      this.$root.apiGETRequest("/facilitytype", this.populateFacilityTypes)
```

Comment:
this api call seems unnecessary, because the same call is happening on the mounted method.  I also have recently put in a pull request to populate facilityTypes in a different way.  I think it was merged into master just recently, so if you pull master into this branch you should see that code.